dev/benchmarks/macrobenchmarks/lib/src/cubic_bezier.dart

```diff
@@ -314,6 +314,9 @@ class AnimatedBezierState extends State<AnimatedBezier>
         vsync: this, duration: const Duration(milliseconds: 1000));
     curve = CurvedAnimation(parent: controller, curve: Curves.linear)
       ..addListener(() {
+        // For App developers:
+        // This is not the suggested approach to implement animation.
+        // AnimatedBuilder is recommended instead fo this.
```

Comment:
@flar I added the comment here. Does this looks fine?

Reply:
I don't think we need to call out app developers, the statement applies regardless of who is reading the code. There is also a typo there.

I'd go with:

```
    // Animations are typically implemented using the AnimatedBuilder widget.
    // This code uses a manual listener for historical reasons and will remain in order to
    // preserve compatibility with the history of measurements for this benchmark.
```

A similar use of `addListener` also appears in the cull_opacity benchmark.